src/components/entry/EntryDetails.js

```diff
@@ -1,19 +1,204 @@
+  margin-top: ${(props) => (props.isDesktop ? '0px' : '90px')};
+`
+
+const ImageContainer = styled.img`
+  width: 100%;
+`
+
+// Wraps all text in the container
+const EntryDetailsContent = styled.div`
+  padding: 23px;
+  box-sizing: border-box;
+`
+
+const PlantName = styled.h2`
+  margin-top: 0px;
+  margin-bottom: 0px;
+  color: #333333;
+`
+
+const ScientificName = styled.small`
+  font-style: italic;
+`
+
+// Wraps the plant name and scientific name, as well as an icon button for desktop
+const HeaderContainer = styled.header`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`
+
+// Wraps description, last updated text, and review and report buttons
+const DescriptionContainer = styled.section`
+  & > *:not(:last-child) {
+    margin-bottom: 14px;
+  }
+`
+
+const Description = styled.p`
+  color: #5a5a5a;
+`
+
+const UpdateText = styled.p`
+  font-size: 14px;
+  font-style: italic;
+`
+
+const ButtonSpacing = styled.div`
+  button {
+    margin-right: 14px;
+  }
+`
+
+const ResourceHeader = styled.h3`
+  color: #333333;
+`
+
+const ResourceLink = styled.a`
+  font-size: 16px;
+`
+
+const ResourceImage = styled.img`
+  height: 20px;
+  width: 25px;
+`
+
+// Wraps all resource images and their links
+const IndividualResourceContainer = styled.small`
+  & > *:not(:last-child) {
+    margin-bottom: 14px;
+  }
+  display: flex;
+  column-count: 2;
+  column-gap: 12px;
+`
+
+const EntryDetails = ({ isDesktop }) => {
   const {
     params: { id },
   } = useRouteMatch()
 
-  return (
-    <div>
-      <br />
-      <br />
-      <br />
-      <br />
-      <br />
-      <p>EntryDetails for id: {id}</p>
-    </div>
+  const [locationData, setLocationData] = useState()
+  const [locationTypeData, setLocationTypeData] = useState()
+
+  useEffect(() => {
+    async function fetchEntryDetails() {
+      const locationData = await getLocationById(id)
+      setLocationData(locationData)
+
+      const locationTypeData = await getTypeById(locationData.type_ids[0])
+      setLocationTypeData(locationTypeData)
+    }
+    fetchEntryDetails()
+  }, [id])
+
+  const handleMapButtonClick = () => {
+    // TODO: handle map button click
+    console.log('Map Button Clicked')
+  }
+
+  const resources = RESOURCES.map(
+    ({ title, urlFormatter = (url) => url, urlKey, icon, iconAlt }) =>
+      locationTypeData?.[urlKey] && (
+        <IndividualResourceContainer>
+          <ResourceImage src={icon} alt={iconAlt} />
+          <ResourceLink
+            target="_blank"
+            rel="noopener noreferrer"
+            href={urlFormatter(locationTypeData[urlKey])}
+          >
+            {title}
+          </ResourceLink>
+        </IndividualResourceContainer>
+      ),
+  )
+
+  return locationData && locationTypeData ? (
+    <EntryDetailsPageContainer isDesktop={isDesktop}>
+      {locationData.photos.length > 0 && (
+        <ImageContainer src={locationData.photos[0].photo.original} alt="" />
```

Comment:
where the alt text at. #a11y

Reply:
Yes, it's best to add an alt text here.

Reply:
Doing it now

Reply:
We can do this in a later PR though. Can you add a TODO here?